src/test/config/bitbucket/helm_parameters

```diff
@@ -4,7 +4,7 @@ CHART_TEST_VALUES_BASEDIR=${chart.testValues.basedir}
 PRODUCT_NAME=bitbucket
 DB_NAME=bitbucket
 POSTGRES_CHART_VERSION=11.6.2
```

Comment:
Should I also be bumping this value to the latest bugfix? I wasn't sure if I should make it different compared to all of the other products, which use 11.6.2